src/BlazorUI/Bit.BlazorUI.Extras/Components/RichTextEditor/@types/quill.d.ts

```
@@ -0,0 +1,195 @@
+        readonly USER: "user";
+    };
+    static version: string;
+    static imports: Record<string, unknown>;
+    static debug(limit: DebugLevel | boolean): void;
+    static find(node: Node, bubble?: boolean): Blot | Quill | null;
+    static import(name: 'core/module'): typeof Module;
+    static import(name: `themes/${string}`): typeof Theme;
+    static import(name: 'parchment'): any;
+    static import(name: 'delta'): typeof Delta;
+    static import(name: string): unknown;
+    static register(targets: Record<string, RegistryDefinition | Record<string, unknown> | Theme | Module | Function>, overwrite?: boolean): void;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)